ci/jira_query.py

```diff
@@ -27,7 +27,7 @@
 # valid.  We've never checked/enforced these before so there have been a lot of values used in the
 # past.
 VALID_COMPONENTS = ('agent', 'build', 'ci', 'csum', 'doc', 'gha', 'il', 'md', 'mercury', 'swim',
-                    'test', 'tools')
+                    'test', 'tools', 'csum')
```

Comment:
This is on L29 now, you had a conflict with master because I added csum to this list (and sorted it alphabetically).